src/modules/Sidebar/Sidebar.js

```diff
@@ -41,6 +42,12 @@ class Sidebar extends Component {
 
     /** Sidebar width. */
     width: PropTypes.oneOf(['very thin', 'thin', 'wide', 'very wide']),
+
+    /** Controls whether or not to close sidebar when click outside */
+    closable: PropTypes.bool,
```

Comment:
Again considering the core behavior, we should actually default this to `true`.  Since it is auto controlled state, it needs to be set in the constructor.